assets/javascripts/discourse/components/gamification-leaderboard.js

```diff
@@ -2,9 +2,19 @@ import Component from "@ember/component";
 import discourseComputed from "discourse-common/utils/decorators";
 import { action } from "@ember/object";
 import showModal from "discourse/lib/show-modal";
+import LoadMore from "discourse/mixins/load-more";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 
-export default Component.extend({
+// Needs to be updated alongside constant in gamification_leaderboard.rb
+const USER_LIMIT = 100;
```

Comment:
Where is this used?

Reply:
https://github.com/discourse/discourse-gamification/pull/20/files#diff-f15608bdf009f80365afaad8f91179b05bfa5b1bdc87d411e5f47f307abaaf0fR63